bin/node/runtime/pangolin/src/pallets/treasury.rs

```diff
@@ -1,57 +1,50 @@
 // --- substrate ---
 use frame_support::PalletId;
-use sp_runtime::{Percent, Permill};
+use sp_runtime::Permill;
 // --- darwinia ---
 use crate::*;
-use darwinia_treasury::{weights::SubstrateWeight, Config};
+use pallet_treasury::{weights::SubstrateWeight, Config, Instance2 as KtonTreasuryInstance};
 
 frame_support::parameter_types! {
 	pub const TreasuryPalletId: PalletId = PalletId(*b"da/trsry");
 	pub const ProposalBond: Permill = Permill::from_percent(5);
-	pub const RingProposalBondMinimum: Balance = 20 * COIN;
-	pub const KtonProposalBondMinimum: Balance = 20 * COIN;
+	pub const ProposalBondMinimum: Balance = 20 * COIN;
 	pub const SpendPeriod: BlockNumber = 3 * MINUTES;
 	pub const Burn: Permill = Permill::from_percent(0);
-	pub const TipCountdown: BlockNumber = 3 * MINUTES;
-	pub const TipFindersFee: Percent = Percent::from_percent(20);
-	pub const TipReportDepositBase: Balance = 1 * COIN;
-	pub const DataDepositPerByte: Balance = 1 * MILLI;
-	pub const BountyDepositBase: Balance = 1 * COIN;
-	pub const BountyDepositPayoutDelay: BlockNumber = 3 * MINUTES;
-	pub const BountyUpdatePeriod: BlockNumber = 3 * MINUTES;
-	pub const MaximumReasonLength: u32 = 16384;
-	pub const BountyCuratorDeposit: Permill = Permill::from_percent(50);
-	pub const BountyValueMinimum: Balance = 2 * COIN;
 	pub const MaxApprovals: u32 = 100;
 }
 
+// In order to use `Tips`, which bounded by `pallet_treasury::Config` rather `pallet_treasury::Config<I>`
+// Still use `DefaultInstance` here instead `Instance1`
 impl Config for Runtime {
 	type PalletId = TreasuryPalletId;
-	type RingCurrency = Ring;
-	type KtonCurrency = Kton;
+	type Currency = Ring;
 	type ApproveOrigin = ApproveOrigin;
 	type RejectOrigin = EnsureRootOrMoreThanHalfCouncil;
-	type Tippers = PhragmenElection;
-	type TipCountdown = TipCountdown;
-	type TipFindersFee = TipFindersFee;
-	type TipReportDepositBase = TipReportDepositBase;
-	type DataDepositPerByte = DataDepositPerByte;
 	type Event = Event;
-	type OnSlashRing = Treasury;
-	type OnSlashKton = Treasury;
+	type OnSlash = Treasury;
 	type ProposalBond = ProposalBond;
-	type RingProposalBondMinimum = RingProposalBondMinimum;
-	type KtonProposalBondMinimum = KtonProposalBondMinimum;
+	type ProposalBondMinimum = ProposalBondMinimum;
 	type SpendPeriod = SpendPeriod;
 	type Burn = Burn;
-	type BountyDepositBase = BountyDepositBase;
-	type BountyDepositPayoutDelay = BountyDepositPayoutDelay;
-	type BountyUpdatePeriod = BountyUpdatePeriod;
-	type MaximumReasonLength = MaximumReasonLength;
-	type BountyCuratorDeposit = BountyCuratorDeposit;
-	type BountyValueMinimum = BountyValueMinimum;
-	type RingBurnDestination = ();
-	type KtonBurnDestination = ();
+	type BurnDestination = Society;
+	type WeightInfo = SubstrateWeight<Runtime>;
+	type SpendFunds = Bounties;
 	type MaxApprovals = MaxApprovals;
+}
+impl Config<KtonTreasuryInstance> for Runtime {
+	type PalletId = TreasuryPalletId;
```

Comment:
For different instances, do we need to choose a different pallet id?

Reply:
No need to. A signle account can handle 2 currencies.